### PR TITLE
Strings: Allow to specify separator in webalize()

### PR DESCRIPTION
**Pull request by legendik**
It's helpful if you need remove accents from string and change whitespace separator to something else then `_`.

### CODE REVIEW

**Comment by Budry** [comment]
:+1:

**Comment by fprochazka** [comment]
I like it!

File: src/Utils/Strings.php [hunk 1]
```diff
@@ -196,17 +196,18 @@ public static function toAscii($s)
 	 * Converts to web safe characters [a-z0-9-] text.
 	 * @param  string  UTF-8 encoding
 	 * @param  string  allowed characters
+	 * @param  string  Whitespace separator
```
Comment (Majkl578) [inline src/Utils/Strings.php]: Whitespace?

**Comment by Budry** [comment]
:+1:

**Comment by dg** [comment]
Why?

**Comment by fprochazka** [comment]
@dg because this

```php
$s = Strings::webalize('1 2 3', NULL, TRUE, '_');
```

is nicer than

```php
$s = str_replace('-', '_', Strings::webalize('1 2 3'));
```

**Comment by dg** [comment]
Why it is useful?

**Comment by JanTvrdik** [comment]
The usecase is too narrow.

**Comment by hrach** [comment]
`$s = Strings::webalize('1 2 3', NULL, TRUE, '_');` is totally criptic, mainly because od the two boolean parameters.

**Comment by dg** [comment]
This is edge case and should be done via a different function.

**Pull request closed without merging**